client/app/scripts/superdesk-archive/tests/archive_spec.js

```diff
@@ -22,7 +22,7 @@ describe('content', function() {
 
     describe('multi service', function() {
         it('can reset on route change', inject(function(multi, $rootScope) {
-            multi.toggle({_id: 1});
+            multi.toggle({_id: 1, selected: true});
```

Comment:
This is a bit confusing: toggle should switch the selection state. If you want to set a selection state given by you create a separate method.